modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -1738,37 +1847,56 @@ the optimization. If CALIB_USE_INTRINSIC_GUESS is set, the coefficient from the
 supplied distCoeffs matrix is used. Otherwise, it is set to 0.
 @param criteria Termination criteria for the iterative optimization algorithm.
 
-The function estimates transformation between two cameras making a stereo pair. If you have a stereo
-camera where the relative position and orientation of two cameras is fixed, and if you computed
-poses of an object relative to the first camera and to the second camera, (R1, T1) and (R2, T2),
-respectively (this can be done with solvePnP ), then those poses definitely relate to each other.
-This means that, given ( \f$R_1\f$,\f$T_1\f$ ), it should be possible to compute ( \f$R_2\f$,\f$T_2\f$ ). You only
-need to know the position and orientation of the second camera relative to the first camera. This is
-what the described function does. It computes ( \f$R\f$,\f$T\f$ ) so that:
+The function estimates the transformation between two cameras making a stereo pair. If one computes
+the poses of an object relative to the first camera and to the second camera,
+( \f$R_1\f$,\f$T_1\f$ ) and (\f$R_2\f$,\f$T_2\f$), respectively, for a stereo camera where the
+relative position and orientation between the two cameras are fixed, then those poses definitely
+relate to each other. This means, if the relative position and orientation (\f$R\f$,\f$T\f$) of the
+two cameras is known, it is possible to compute (\f$R_2\f$,\f$T_2\f$) when (\f$R_1\f$,\f$T_1\f$) is
+given. This is what the described function does. It computes (\f$R\f$,\f$T\f$) such that:
 
 \f[R_2=R*R_1\f]
-\f[T_2=R*T_1 + T,\f]
+\f[T_2=R*T_1 + T.\f]
```

Comment:
Not related to this specific change. I have already seen this before.

```suggestion
\f[T_2=R \times T_1 + T.\f]
```

`*` means usually the convolution operation.

The same changes should be done everywhere, so it is better to keep it like this.

Reply:
now that I look at it, shouldn't we simply do:

```
\f[R_2=R R_1\f]
\f[T_2=R T_1 + T.\f]
```

Reply:
?

Reply:
Yes, it is simpler like that.